src/main/java/duke/storage/CsvConverter.java

```diff
@@ -0,0 +1,103 @@
+package duke.storage;
+
+import static duke.util.Keyword.CSV_SEPARATOR;
+import static duke.util.Keyword.DATE_TIME_FORMAT;
+import static duke.util.Keyword.KEYWORD_DEADLINE;
+import static duke.util.Keyword.KEYWORD_EVENT;
+import static duke.util.Keyword.KEYWORD_TODO;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exception.InvalidFileFormatException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Converts the task in CSV format to a Task object.
+ */
+public class CsvConverter {
+
+    private static final String DONE = "done";
+    private static final String NOT_DONE = "not done";
+
+    /**
+     * Creates a task from the CSV format of the task.
+     *
+     * @param input CSV format of task.
+     * @return Task corresponding to the csv string.
+     */
+    protected static Task parseToTask(String input) throws InvalidFileFormatException {
+        String[] resultArr = parseString(input);
+
```

Comment:
is there a reason for this extra space? If it is in the java coding standard do tell me too!

Reply:
Fixed